packages/mask/src/plugins/IdeaMarket/hooks/useFetchIdeaTokens.ts

```diff
@@ -0,0 +1,18 @@
+import { useAsync } from 'react-use'
+import { fetchTwitterLookups } from '../apis'
+import { PluginIdeaMarketRPC } from '../messages'
+
+export function useFetchIdeaTokensBySearch(searchText: string, page: number, filters: string[]) {
+    const { value, error, loading } = useAsync(async () => {
+        const tokens = await PluginIdeaMarketRPC.fetchAllTokens(searchText, page, filters)
```

Comment:
@guanbinrui Used RPC bridge instead of frontend call but it's not working... it doesn't make sense. Can you help me on this one ?